manifests/modules/automation/gitops/flux/buildspec-manifest.yml

```diff
@@ -0,0 +1,36 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - echo Build started on `date`
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR in $AWS_REGION
+      - COMMIT_HASH=$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | cut -c 1-8)
+      - IMAGE_TAG=i$(date +%Y%m%d%H%M%S)-${COMMIT_HASH:=latest}
+      - echo ECR_URI=$ECR_URI
+      - echo COMMIT_HASH=$COMMIT_HASH
+      - echo IMAGE_TAG=$IMAGE_TAG
+      - aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $ECR_URI
+  build:
+    commands:
+      - echo Building the Docker manifest...
+      # Based on the Docker documentation, must include the DOCKER_CLI_EXPERIMENTAL environment variable
+      # https://docs.docker.com/engine/reference/commandline/manifest/
+      - export DOCKER_CLI_EXPERIMENTAL=enabled
+      - docker manifest create $ECR_URI:$IMAGE_TAG $ECR_URI:latest-arm64 $ECR_URI:latest-amd64
+      - docker manifest create $ECR_URI:latest $ECR_URI:latest-arm64 $ECR_URI:latest-amd64
```

Comment:
One is for tag latest, which I use for the initial image in manifest, when the app is deployed.
Another is for tag iTtimestapm, which is used for incremental builds and FluxImageUpdater.
ImageManifest combine together reference to latest-arm64 and latest-amd64 images.

Reply:
latest is not a best practices to uses, do we uses the latest somewhere after ?

Reply:
We don't use latest in deployment, it is rather a label to show that this image with tag iTimestapm is the latest. We can comment out or remove lines which use/build latest. But sometimes it is useful to have it, because it can be used as a trigger in the CodePipeline to start some other actions when a new image has been built.